src/pages/case-studies/[slug].astro

```diff
@@ -0,0 +1,175 @@
+> -->
+<Layout title={data.title} description={data.description} pageType="article">
+	<!-- {
+		coverImage ? (
+			<div class="mx-auto w-full max-w-screen-lg md:-mb-10 lg:-mb-12 xl:-mb-16">
+				<img
+					src={coverImage.src}
+					width={coverImage.width}
+					height={coverImage.height}
+					loading="lazy"
+					decoding="async"
+					alt=""
+					class="w-full"
+				/>
+			</div>
+		) : (
+			<div
+				class:list={[
+					"w-full h-64 -mb-56 md:h-64 md:-mb-52 lg:h-96 lg:-mb-80 noise-underlay",
+					placeholderGradient,
+				]}
+			/>
+		)
+	} -->
+	<div
+		class:list={[
+			"w-full h-64 -mb-56 md:h-64 md:-mb-52 lg:h-96 lg:-mb-80 noise-underlay",
+			placeholderGradient,
+		]}
+	>
+	</div>
+	{
+		headerImage && (
+			<img
+				src={headerImage.src}
+				width={headerImage.width}
+				height={headerImage.height}
+				loading="lazy"
+				decoding="async"
+				alt=""
+				class="mx-auto mb-20 mt-8 max-w-full"
+			/>
+		)
+	}
+
+	<div class="grid-container relative mx-auto max-w-screen-lg md:z-10">
+		<div
+			class="noise-container mx-auto w-fit max-w-full bg-astro-gray-700 py-5 sm:py-8 md:px-10 md:py-12 lg:p-12 lg:py-16 xl:p-16 xl:py-20"
+		>
+			<div class="noise"></div>
+			<header class="mx-auto w-full max-w-4xl text-center">
+				<p class="code text-astro-gray-200">
+					Case Study &bull;
+					<time datetime={data.publishDate.toISOString()}>
+						{format(data.publishDate, "MMMM d, yyyy")}
+					</time>
+				</p>
+				<h1 class="mb-4 mt-2 flex flex-col items-center sm:mb-5 sm:mt-3">
+					<strong class="heading-3 mb-2 block font-normal md:heading-2">{data.title}</strong>
+					<div class="heading-4 max-w-2xl font-normal leading-tight md:heading-3">
+						{data.description}
+					</div>
+				</h1>
+				<div class="code flex flex-wrap items-center justify-center gap-4 text-astro-gray-200">
+					<span>By</span>
+					{
+						authors.map((author) => {
+							return (
+								<div class="flex items-center gap-2">
+									<Avatar name={author.slug} lg />
+									<span>{author.data.name}</span>
+								</div>
+							)
+						})
+					}
+				</div>
+			</header>
+			<hr class="-mx-16 my-16 border-astro-gray-500" />
+			<article class="prose">
+				<Content />
+			</article>
+			<footer
+				class="mx-auto flex max-w-prose flex-col items-center gap-8 overflow-hidden rounded-lg border border-astro-purple bg-astro-gray-700 px-8 md:flex-row"
+			>
+				<div class="flex flex-1 flex-col gap-2 pt-8 text-center md:my-5 md:pt-0 md:text-left">
+					<h2 class="heading-4">Take Astro for a spin</h2>
+					<p class="text-sm">Discover what our flexible framework can do for you!</p>
+				</div>
+				<a
+					href="https://docs.astro.build"
+					target="_blank"
+					rel="noopener noreferrer"
+					class="button button-primary button-sm">Get Started</a
+				>
+				<img
+					src={houstonWink.src}
+					alt=""
+					class="order-first -mb-px h-24 w-36 object-contain object-bottom md:self-end"
+				/>
+			</footer>
```

Comment:
Looks like this could do with some love on mobile:
<img width="349" alt="image" src="https://github.com/withastro/astro.build/assets/357379/56c77582-1288-448f-b21e-9f2f087963fc">

Off the top of my head: flip Houston down to the bottom and add some `py`?

Reply:
Yes! Agree with Chris. If we can move him to the bottom, that would be great if not I sent you on Discord an uncropped version of Houston that we can use in this layout you have. 

Comments on this current layout for mobile:
- Tighten up the spacing between elements (Houston, heading, text, button)
- Add top and bottom padding to give some breathing room

Let's make it look more like this:
![image](https://github.com/withastro/astro.build/assets/17228655/ae4a574a-c1e4-4e05-97b4-db9b476dbc4f)

If we can move Houston flush to the bottom, that would be lovely, like in Astro.new. If not, swap out for the uncropped image. 

[houston_wink-cta.zip](https://github.com/withastro/astro.build/files/11826652/houston_wink-cta.zip)

Reply:
Strange, Houston was _supposed_ to be on the bottom here, but I guess that broke somehow 🫠